docs/software-requirements.rst

```diff
@@ -2,20 +2,6 @@
 Software Requirements
 =====================
 
-**Operating System**
```

Comment:
I would leave the Operating System requirements in. Any flavor of linux is probably fine and possibly OS X too, but we haven't tested on anything else.  And we know there would be issues on windows.